tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -3220,12 +3756,28 @@ TEST_F(HVACVRFFixture, VRFTest_SysCurve)
     FinalZoneSizing(CurZoneEqNum).CoolDesTemp = 13.1;                   // 55.58 F
     FinalZoneSizing(CurZoneEqNum).CoolDesHumRat = 0.009297628698818194; // humrat at 12.77777 C db / 12.6 C wb
 
-    SimulateVRF(
-        VRFTU(VRFTUNum).Name, CurZoneNum, FirstHVACIteration, SysOutputProvided, LatOutputProvided, ZoneEquipList(CurZoneEqNum).EquipIndex(EquipPtr));
+    bool HeatingActive = false;
+    bool CoolingActive = false;
+    int OAUnitNum = 0;
+    Real64 OAUCoilOutTemp = 0.0;
+    bool ZoneEquipment = true;
+    DataAirLoop::AirLoopInputsFilled = true;
+
+    SimulateVRF(VRFTU(VRFTUNum).Name,
+                FirstHVACIteration,
+                CurZoneNum,
+                ZoneEquipList(CurZoneEqNum).EquipIndex(EquipPtr),
+                HeatingActive,
+                CoolingActive,
+                OAUnitNum,
+                OAUCoilOutTemp,
+                ZoneEquipment,
+                SysOutputProvided,
+                LatOutputProvided);
 
     ASSERT_EQ(1, NumVRFCond);
-    ASSERT_EQ(ZoneInletAirNode,
-              ZoneEquipConfig(VRFTU(VRFTUNum).ZoneNum).InletNode(1)); // only 1 inlet node specified above in ZoneHVAC:EquipmentConnections
+    ASSERT_EQ(TUInletAirNode,
+              ZoneEquipConfig(VRFTU(VRFTUNum).ZoneNum).ExhaustNode(1)); // only 1 exhaust node specified above in ZoneHVAC:EquipmentConnections
```

Comment:
This one just blows my mind. The old GetVRFTUZoneInletAirNode function gets the TU inlet node, not the zone inlet node. So how could the TU inlet node ever be equal to the zone inlet node? The TU inlet node MUST be equal to the zone exhaust node. I renamed that function to GetVRFTUInletAirNode to be more representative of the function. I guess I need to run this in develop to see what this was doing and why the TU inlet node was equal to the zone inlet node.

Reply:
I've figured this out. Will revert because system reports require the zone inlet node.